Gemfile

```diff
@@ -57,6 +57,7 @@ gem "csv"
 gem "logger"
 gem "ostruct"
 
+gem "bops_consultees", path: "engines/bops_consultees"
```

Comment:
Incredibly tiny nitpick but I'm not sure what's going on with the ordering here: maybe it was intended to be `core` then all the others alphabetically or something?

Reply:
just did alphabetically but this makes sense